test/Test/Sync.hs

```diff
@@ -33,3 +39,16 @@ testBadGitHubIssueUrl1 = GitHub.URL "https://api.github.com/randomWord/owner123/
 
 testBadGitHubIssueUrl2 :: GitHub.URL
 testBadGitHubIssueUrl2 = GitHub.URL "api.github.com/repos/owner123/repo123/issues/1"
+
+downloadFileSpec :: AppEnv -> Spec
+downloadFileSpec env = describe "downloadFile" $ do
+    it "should succeed with 200 status code when passed a valid Url" $
+       env & succeeds (downloadFileImpl issueWantedUrl)
+    it "should fail with notFound error when passed a non-existent Url" $
```

Comment:
Let's add one more test using `equals` to compare the content with some file. Just in case, to make sure, that we actually get the proper content 🙂 The following file is the smallest in our repo that I've found:

* https://github.com/kowainik/issue-wanted/blob/master/app/Main.hs